.babelrc

```diff
@@ -1,3 +0,0 @@
-{
-    "plugins": ["lodash"]
-}
```

Comment:
Having a `.babelrc` [totally crashes gatsby v2](https://www.gatsbyjs.org/docs/migrating-from-v1-to-v2/#move-babel-configuration). It is "forbidden" to have a `.babelrc` at the root: